samples/contrib/pytorch-samples/build.sh

```diff
@@ -25,7 +25,7 @@ export full_image_name=$full_image_name
 ## build and push docker - to fetch the latest changes and install dependencies
 # cd pytorch_kfp_components
 
-docker build --no-cache -t $full_image_name -f Dockerfile-cpu .
```

Comment:
We no longer need the dockerfile name as we have only one dockerfile now for both cpu and gpu

Reply:
/lgtm